sds_data_manager/lambda_code/SDSCode/pipeline_lambdas/batch_starter.py

```diff
@@ -389,7 +395,7 @@ def try_to_submit_job(
         "--version",
         version,
         "--dependency",
-        upstream_dependencies,
+        os.path.basename(dependency_file_path),
```

Comment:
If we already have a Path object, we should use the pathlib methods.
```suggestion
        dependency_file_path.name,
```